.github/workflows/changed_files.yml

```diff
@@ -0,0 +1,114 @@
+name: List files changed as GitHub comment
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  list-files-changed:
+    if: github.event.pull_request.head.repo.full_name == github.repository
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 2
+
+      - name: Format list of changed files
+        id: format
+        env:
+          GH_TOKEN: ${{ github.token }}
+        run: |
+          declare -a output added_files modified_files removed_files no_preview_files
+
+          pr_num="${{ github.event.pull_request.number }}"
+
+          all_changed_files="$(gh api repos/wandb/docs/pulls/${pr_num}/files -q '.[] | [.status,.filename] | @tsv' | sort)"
+
+          for changed_file in "${all_changed_files[@]}"; do
+            status="$(echo \"$changed_file\" | awk -F ' ' {'print $1'})"
+            echo "Status: $status"
+            path="$(echo \"$changed_file\" | awk -F ' ' {'print $2'})"
+            echo "Path: $path"
+            if [ "$status" = "added" ]; then
+              added_files+=("$path")
+            elif [ "$status" = "modified" ]; then
+              modified_files+=("$path")
+            elif [ "$status" = "removed" ]; then
+              removed_files+=("path")
+            fi
+          done
+
+          echo "Added:"
+          echo "${added_files[@]}"
+          echo
+
+          echo "Modified:"
+          echo "${modified_files[@]}"
+          echo
+
+          echo "Removed:"
+          echo "${removed_files[@]}"
+          echo
+
+          function generatePreviewURL() {
+            # Preprocess input paths to output paths
+            #   using # as sed separation character
+            #
+            #   - Remove content/
+            #   - Remove _index/
+            #   - Remove integration_tutorials/
+
+            # Generate HTML preview links for added files in content/
+            for file in ${added_files[@]}; do
+              if [[ "$file" =~ .*\/content\/.*\.md ]]; then
+                output_path="$(echo \"$file\" | sed -E 's#content\/##g' | sed -E 's#/_index/##g' | sed -E 's#integration-tutorials\/##g')
```

Comment:
This is missing a closing double quote

Original:
output_path="$(echo \"$file\" | sed -E 's#content\/##g' | sed -E 's#/_index/##g' | sed -E 's#integration-tutorials\/##g')

Fixed:
output_path="$(echo \"$file\" | sed -E 's#content\/##g' | sed -E 's#/_index/##g' | sed -E 's#integration-tutorials\/##g')"